### PR TITLE
[117] Add required attributes to html elements in the form, add new v…

### PR DESCRIPTION
**Pull request by przemyslawhaptas**
…alidations to the Setting model

### CODE REVIEW

**Comment by przemyslawhaptas** [comment]
We could add more validations in the model - e.g. for the fields' presence.

File: app/views/settings/_form.html.erb [hunk 1]
```diff
@@ -2,36 +2,36 @@
...
   <div class="form-field">
     <%= f.label :event_url %>
     <span class="event-url-prefix">http://</span>
-    <%= f.text_field :event_url, class: "event-url-field" %>
+    <%= f.text_field :event_url, class: "event-url-field", required: true %>
   </div>
   <div class="form-field">
-    <%= f.label :available_spots %>
-    <%= f.number_field :available_spots %>
+    <%= f.label :available_spots, min: 0, required: true %>
```
Comment (czorek) [inline app/views/settings/_form.html.erb]: This is a label, not a field :P

File: app/views/settings/_form.html.erb [hunk 2]
```diff
@@ -2,36 +2,36 @@
...
+    <%= f.number_field :available_spots%>
   </div>
   <div class="form-field">
     <%= f.label :required_rates_num %>
-    <%= f.number_field :required_rates_num %>
+    <%= f.number_field :required_rates_num, min: 0, required: true %>
```
Comment (czorek) [inline app/views/settings/_form.html.erb]: Also, shouldn't we reflect the view validations in the model?

**Comment by przemyslawhaptas** [comment]
[[117] 'Settings' form accepts nil in 'Available spots' field](https://trello.com/c/chVDo3GW/82-117-settings-form-accepts-nil-in-available-spots-field)

**Pull request merged**